src/createServer.js

```diff
@@ -1,8 +1,44 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'text/plain');
+
+    const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+
+    if (!normalizedUrl.pathname.startsWith('/file')) {
+      res.statusCode = 400;
+      res.statusMessage = 'Path should start with "/file/"';
```

Comment:
The status message 'Path should start with "/file/"' is slightly misleading since the check is for paths starting with '/file'. Consider updating the message to 'Path should start with "/file"'.